### PR TITLE
Action alias CLI commands and minor consistency changes

### PR DESCRIPTION
**Pull request by Kami**
This pull request adds CLI comments for viewing action aliases (list, get).

In addition to that, I made some other changes so it's consistent with other content pack resources (actions, rules, etc.):

* Also allow user to retrieve action alias by ref
* Update supported query param filters
* Add `enabled` attribute to the ActionAlias model

### CODE REVIEW

File: st2client/st2client/commands/action_alias.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        super(ActionAliasBranch, self).__init__(
+            ActionAlias, description, app, subparsers,
+            parent_parser=parent_parser, read_only=True,
+            commands={
+                'list': ActionAliasListCommand,
+            })
+
+
+class ActionAliasListCommand(resource.ResourceCommand):
+    display_attributes = ['ref', 'pack', 'name', 'description', 'enabled', 'formats']
```
Comment (manasdk) [inline st2client/st2client/commands/action_alias.py]: Can you paste a sample `st2 actionalias list` with actionalias containing multiple formats? If the formatting looks weird there might be value in dropping the formats and only showing them for `st2 actionalias get`.

File: st2client/st2client/commands/action_alias.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+]
+
+
+class ActionAliasBranch(resource.ResourceBranch):
+    def __init__(self, description, app, subparsers, parent_parser=None):
+        super(ActionAliasBranch, self).__init__(
+            ActionAlias, description, app, subparsers,
+            parent_parser=parent_parser, read_only=True,
+            commands={
+                'list': ActionAliasListCommand,
```
Comment (manasdk) [inline st2client/st2client/commands/action_alias.py]: do we also support `get`?

**Comment by manasdk** [comment]
Overall I am ok with the change. Few questions but not blockers -

+1

File: st2client/st2client/commands/action_alias.py (hunk 1 above)
Comment (Kami) [inline st2client/st2client/commands/action_alias.py]: We display arrays as string joined with "," and I think it looks OK, but I'm also fine with dropping it in list.

```bash
python st2client/st2client/shell.py action-alias list
+---------------------------+------+-----------------------+-----------------------------------+---------+-----------------------------------+
| ref                       | pack | name                  | description                       | enabled | formats                           |
+---------------------------+------+-----------------------+-----------------------------------+---------+-----------------------------------+
| st2.st2_actions_list      | st2  | st2_actions_list      | List available StackStorm         |         | st2 list actions, st2 list        |
|                           |      |                       | actions.                          |         | actions from {{ pack }}           |
| st2.st2_executions_get    | st2  | st2_executions_get    | Retrieve details for a single     |         | st2 get execution {{ id }}, st2   |
|                           |      |                       | execution.                        |         | show execution {{ id }}, st2      |
|                           |      |                       |                                   |         | executions get {{ id }}, st2      |
|                           |      |                       |                                   |         | executions show {{ id }}          |
| st2.st2_executions_list   | st2  | st2_executions_list   | List available StackStorm         |         | st2 list executions, st2 list     |
|                           |      |                       | executions.                       |         | executions with status {{ status  |
|                           |      |                       |                                   |         | }}, st2 list executions for       |
|                           |      |                       |                                   |         | action {{ action }}               |
| st2.st2_executions_re_run | st2  | st2_executions_re_run | Re-run an action execution.       |         | st2 re-run execution {{ id }},    |
|                           |      |                       |                                   |         | st2 executions re-run {{ id }}    |
| st2.st2_rules_list        | st2  | st2_rules_list        | List available StackStorm rules.  |         | st2 list rules, st2 list rules    |
|                           |      |                       |                                   |         | from {{ pack }}                   |
| st2.st2_sensors_list      | st2  | st2_sensors_list      | List available StackStorm         |         | st2 list sensors, st2 list        |
|                           |      |                       | sensors.                          |         | sensors from {{ pack }}           |
+---------------------------+------+-----------------------+-----------------------------------+---------+-----------------------------------+
```

File: st2client/st2client/commands/action_alias.py (hunk 2 above)
Comment (Kami) [inline st2client/st2client/commands/action_alias.py]: Yeah, it gets automatically hooked up in the parent class, but for explicitness sake I will also add it here.

File: st2client/st2client/commands/action_alias.py (hunk 1 above)
Comment (Kami) [inline st2client/st2client/commands/action_alias.py]: Decided to drop the attribute in `list` output, now it looks better and more compact.

**Pull request merged**